pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -260,6 +267,16 @@ func (pc *PackageControllerClient) GetCuratedPackagesRegistries() (sourceRegistr
 		if pc.eksaRegion != eksaDefaultRegion {
 			defaultImageRegistry = strings.ReplaceAll(defaultImageRegistry, eksaDefaultRegion, pc.eksaRegion)
 		}
+
+		regionalRegistry := GetRegionalRegistry(defaultRegistry, pc.eksaRegion)
+		if err := pc.registryAccessTester(ctx, pc.eksaAccessKeyID, pc.eksaSecretAccessKey, regionalRegistry, pc.eksaRegion); err == nil {
+			// use regional registry when the above credential is good
+			logger.V(6).Info("Using regional registry")
+			defaultRegistry = regionalRegistry
+			defaultImageRegistry = regionalRegistry
+		} else {
+			logger.V(6).Info("Using non-regional registry", "RegionalRegistryAccessIssue", err)
```

Comment:
What does it mean to use a non-regional registry?

Any reason we don't fail if they can't access a regional registry? Curious if we're going to keep or deprecate the fallback registry indefinitely.

```suggestion
			logger.V(6).Info("Using fallback registry", "Registry", defaultRegistry" "RegionalRegistryAccessIssue", err)
```

Reply:
There will be users still using the old registry, POC or old subscribers. The curated packages would still be installed in this case, because customers can set the AWS secrets after getting subscription.